core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -244,12 +244,14 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     * @param getNewAcls function to transform existing acls to new ACLs
     * @return boolean indicating if a change was made
     */
-  private def updateResourceAcls(resource: Resource)(getNewAcls: Set[Acl] => Set[Acl]): Boolean = {
+  private def updateResourceAcls(resource: Resource, acls: Set[Acl], add: Boolean)(getNewAcls: Set[Acl] => Set[Acl]): Boolean = {
     var currentVersionedAcls =
       if (aclCache.contains(resource))
         getAclsFromCache(resource)
       else
         getAclsFromZk(resource)
+    error(s"updateResourceAcls acls: $acls add : $add , currentVersionedAcls = $currentVersionedAcls")
```

Comment:
In a normal run where test passes, I see this.
Consider changing to debug or info.

Reply:
just added for testing with jenkins job. Added ERROR log level to reduce amount of logs